### PR TITLE
Update readme to mention the github releases

### PR DESCRIPTION
**Pull request by jspaezp**
Hello there!

First off, I am very impressed by the project so I wanted to make a minor contribution to the documentation. I added "installation" instructions for the pre-compiled binaries in the github releases.

In an additional effort to improve documentation:

-  On the "notes" you also mention:
    > - The majority of parameters are optional - only "database.fasta", "precursor_tol", and "fragment_tol" are required. Sage will try and use reasonable defaults for any parameters not supplied
    
    I am assuming that means reasonable for data-dependent narrow-window high-resolution ms2 runs, right? Should that be explicit?

- Should a page be added to explain all the features used internally for re-scoring?

- [x] Document the change in CLI that allows over-writting the json
https://github.com/lazear/sage/pull/19
- [x] Document docker image usage (https://github.com/lazear/sage/pull/18) -> docker pull ghcr.io/lazear/sage:master

Thanks a lot for the great project!
Sebastian

### CODE REVIEW

**Comment by lazear** [comment]
Hi Sebastian,

Thanks for the comments and PR - there should definitely be some effort to expand on the documentation!

1. Sure, we could add mention that Sage only supports DDA runs. I think most of the defaults are also reasonable for low-res MS2 scans as well, since precursor/fragment tolerances are required parameters - the other options controling database generation & spectral processing should largely be independent of MS2 resolution (tweaking them can lead to optimal searching however - e.g. turning on deisotoping for high-res scans).
2. If you'd like to get started on this, you can reference https://github.com/lazear/sage/blob/f5d065012dbd62f7cd8b01f020909d9f4db36b02/src/lda/mod.rs#L169-L188 and https://lazear.github.io/sage/#lda 
3. & 4 I am also happy to merge in, if you provide! Perhaps we can utilize the GitHub wiki feature?

Thanks,
Mike

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Sage has excellent performance characteristics (5x faster than - the closed sour
 
 # Installation
 
+## Installing the development version
```
Comment (lazear) [inline README.md:46]: I wonder if "Installation" is the proper terminology to use here (I realize I used that as the heading...). Perhaps "Compiling the development version"?

File: README.md [hunk 2]
```diff
@@ -56,12 +58,56 @@ cd sage
 cargo run --release tests/config.json 
 ```
 
+## Installing the latest release
```
Comment (lazear) [inline README.md:61]: Likewise, "Downloading the latest release"

File: README.md [hunk 3]
```diff
@@ -56,12 +58,56 @@ cd sage
...
+
+1. The paths to the raw mzML data
+2. The path to the database (fasta file)
+3. The output directory
+
+like so ...
+
+```
+# specify fasta and output dir:
+sage -f proteins.fasta -o cool_project config.json
```
Comment (lazear) [inline README.md:105]: Would you change this to `sage -f proteins.fasta -o output_directory config.json`?

File: README.md [hunk 4]
```diff
@@ -74,6 +120,8 @@ Two notes:
 - Tolerances are specified on the *experimental* m/z values. To perform a -100 to +500 Da open search (mass window applied to *precursor*), you would use `"da": [-500, 100]`
 
 ```jsonc
+// Note that json does not allow comments, they are here just as explaination
```
Comment (lazear) [inline README.md:123]: Typo: explanation

**Review by lazear — COMMENTED** [review]
Made a few comments, overall looks good - thanks!

File: README.md (hunk 4 above)
Comment (jspaezp) [reply README.md:123]: mb, not my native language

File: README.md (hunk 3 above)
Comment (jspaezp) [reply README.md:105]: aggree

File: README.md (hunk 2 above)
Comment (jspaezp) [reply README.md:61]: done

File: README.md (hunk 1 above)
Comment (jspaezp) [reply README.md:46]: I think that even though it is not 100% accurate, it is a good "you know what i mean" expression

**Review by lazear — APPROVED** [review]

**Pull request merged**